shop_manager.rb

```diff
@@ -0,0 +1,80 @@
+require_relative "lib/item_repository"
+require_relative "lib/order_repository"
+
+
+class Application
+  def initialize(database, io, repo1, repo2)
+    DatabaseConnection.connect(database)  # database name might vary from a device to another
+    @io = io
+    @repo1 = repo1
+    @repo2 = repo2
```

Comment:
You could change the names of repo1 / repo 2 to define which repo they are referring to - items/orders